tests/expectedSchemas/booksChangeStateManySchema.js

```diff
@@ -373,6 +373,7 @@ module.exports = {
                   },
                 },
               },
+              'additionalProperties': false,
```

Comment:
Is there a schema where `additionaProperties` on array is set to `true`?

Reply:
You mean like [this](https://github.com/mia-platform/crud-service/blob/e4bc6b4ead426d5e3801fdc82f77a49b7debc0e8/tests/collectionDefinitions/books.js#L170)

Reply:
OK, there is [this](https://github.com/mia-platform/crud-service/blob/e4bc6b4ead426d5e3801fdc82f77a49b7debc0e8/tests/collectionDefinitions/books.js#L161).